packages/vue/src/index.ts

```diff
@@ -105,7 +105,7 @@ const Frontegg: PluginObject<PluginOptions> | any = {
     fronteggApp = initialize({
       ...rest,
       onRedirectTo,
-      basename: router?.options.base,
+      basename: getRouterBaseName(router),
```

Comment:
pay attention that here they used optional for router.. they do it also in line 104. maybe worth adding `?` to the helper function you created

Reply:
I already did